### PR TITLE
Add warning when Teletraan user tries to save new config

### PR DESCRIPTION
**Pull request by liyaqin1**
Teletraan to display config change and let user confirm when config change is saved

### CODE REVIEW

**Comment by liyaqin1** [comment]
Screenshot 2023-04-04 at 7 20 58 PM

File: deploy-board/deploy_board/templates/configs/config_map.tmpl [hunk 1]
```diff
@@ -42,6 +42,30 @@
...
+            <form id="saveEntryFormId" class="form-horizontal" method="post" role="form">
+                <div class="modal-header">
+                    <button type="button" class="close" data-dismiss="modal"><span
+                            aria-hidden="true">&times;</span><span class="sr-only">Close</span>
+                    </button>
+                    <h4 class="modal-title" id="saveEnvModalLabel">Save New {{ panelTitle }}</h4>
+                </div>
+                <div class="modal-body">
+                    Are you sure you want to <strong>save new config</strong> for this cluster?<br>
+                    Please double check the new added config with red color before saving. Thanks.
```
Comment (ntascii) [inline deploy-board/deploy_board/templates/configs/config_map.tmpl:58]: red color means new entries or error entries? If it's new probably we should have them in different color. Red often means error I guess.

Comment (liyaqin1) [reply deploy-board/deploy_board/templates/configs/config_map.tmpl:58]: new entries. what's the best color for new entries on your mind please?

Comment (ntascii) [reply deploy-board/deploy_board/templates/configs/config_map.tmpl:58]: not an UX expert but probably blue green or yellow.

**Comment by liyaqin1** [comment]
Current behavior:
After adding, it is not easy to figure out what changed. When saving, there is no confirmation box.

PR behavior:
After adding, the new item is in yellow color. When saving, there is a confirmation box to ask user to double check.

File: deploy-board/deploy_board/templates/configs/config_map.tmpl (hunk 1 above)
Comment (liyaqin1) [reply deploy-board/deploy_board/templates/configs/config_map.tmpl]: ok, used yellow ^^

**Comment by ntascii** [comment]
thanks.

**Review by ntascii — APPROVED** [review]

**Pull request merged**